tests/src/merkle_proof/tests/functions/process_proof.rs

```diff
@@ -29,7 +27,7 @@ mod success {
     async fn processes_merkle_proof() {
         let instance = merkle_proof_instance().await;
 
-        let depth = 8;
+        let depth = 16;
```

Comment:
should we increase depth on any of the other tests?  Doesn't have to be in this pr but something to consider.

Reply:
I've tested it with depth of 24, which takes about 10 minutes to create the tree on my local machine. This would take way too long for CI